parity/cli/mod.rs

```diff
@@ -294,7 +294,7 @@ usage! {
 
 			ARG arg_chain: (String) = "foundation", or |c: &Config| c.parity.as_ref()?.chain.clone(),
 			"--chain=[CHAIN]",
-			"Specify the blockchain type. CHAIN may be either a JSON chain specification file or olympic, frontier, homestead, mainnet, morden, ropsten, classic, expanse, musicoin, ellaism, easthub, testnet, kovan or dev.",
+			"Specify the blockchain type. CHAIN may be either a JSON chain specification file or olympic, frontier, homestead, mainnet, morden, ropsten, classic, expanse, musicoin, ellaism, easthub, ethereumsocial, testnet, kovan or dev.",
```

Comment:
this is growing out of hand. we need some idea here.

Reply:
Yeah this is not sustainable, maybe put some basics in there and then a reference to a full list somewhere else.